src/pudl/analysis/timeseries_cleaning.py

```diff
@@ -1,11 +1,13 @@
 """Screen timeseries for anomalies and impute missing and anomalous values.
 
+For a narrative discussion of these methods aimed at data users, see
+:doc:`/methodology/timeseries_imputation`.
+
 The screening methods were originally designed to identify unrealistic data in the
-electricity demand timeseries reported to EIA on Form 930, and have also been
-applied to the FERC Form 714, and various historical demand timeseries
-published by regional grid operators like MISO, PJM, ERCOT, and SPP.
+electricity demand timeseries reported in :doc:`/data_sources/eia930`, and we have also
+applied them to demand data from :doc:`/data_sources/ferc714`.
 
-They are adapted from code published and modified by:
+Screening methods are adapted from code published and modified by:
 
 * Tyler Ruggles <truggles@carnegiescience.edu>
```

Comment:
Might align this list with the one in docs -- add Alicia Wangle and David Farnham, maybe switch to GitHub repo links rather than emails?

Reply:
Updated!